src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt

```diff
@@ -16,36 +19,66 @@ import org.picocontainer.Disposable
 import tanvd.grazi.GraziConfig
 import tanvd.grazi.ide.ui.rules.GraziRulesTree
 import tanvd.grazi.language.Lang
-import java.awt.Desktop
 import javax.swing.JComponent
 import javax.swing.JEditorPane
-import javax.swing.event.HyperlinkEvent
+import javax.swing.JLabel
 
 class GraziSettingsPanel : ConfigurableUi<GraziConfig>, Disposable {
     private val cbEnableGraziSpellcheck = JBCheckBox(msg("grazi.ui.settings.enable.text"))
     private val cmbNativeLanguage = ComboBox<Lang>(Lang.sortedValues.toTypedArray())
-    private val adpEnabledLanguages by lazy { GraziAddDeleteListPanel() }
 
-    private val descriptionPane = JEditorPane().apply {
+    private val ruleLink = LinkLabel<Any?>(msg("grazi.ui.settings.rules.rule.description"), null)
+    private val linkPanel = panel(HorizontalLayout(0)) {
+        border = padding(JBUI.insetsBottom(10))
+        isVisible = false
+
+        add(ruleLink)
+        add(JLabel(AllIcons.Ide.External_link_arrow))
+    }
+
+    private val smallInfoPane = JEditorPane().apply {
```

Comment:
We've got two `JEditorPane().apply { ... }`. Why not to create `pane { ... }` for them in DSL? :)

Reply:
Done